slm.lock

```diff
@@ -1,4 +1,9 @@
-semver={locator="StanzaOrg/semver",version="0.1.6",hash="13f3eba0e1cfc5bf926134ef8b43bd7d07981ea4"}
+libcurl={version="8.6.0",type="conan",options="[\"fPIC\" => \"True\" \"shared\" => \"False\" \"with_brotli\" => \"False\" \"with_c_ares\" => \"False\" \"with_ca_bundle\" => \"auto\" \"with_ca_fallback\" => \"False\" \"with_ca_path\" => \"auto\" \"with_cookies\" => \"True\" \"with_crypto_auth\" => \"True\" \"with_dict\" => \"True\" \"with_docs\" => \"False\" \"with_file\" => \"True\" \"with_ftp\" => \"True\" \"with_gopher\" => \"True\" \"with_http\" => \"True\" \"with_imap\" => \"True\" \"with_ipv6\" => \"True\" \"with_largemaxwritesize\" => \"False\" \"with_ldap\" => \"False\" \"with_libgsasl\" => \"False\" \"with_libidn\" => \"False\" \"with_libpsl\" => \"False\" \"with_librtmp\" => \"False\" \"with_libssh2\" => \"False\" \"with_mqtt\" => \"True\" \"with_nghttp2\" => \"False\" \"with_ntlm\" => \"True\" \"with_ntlm_wb\" => \"True\" \"with_pop3\" => \"True\" \"with_proxy\" => \"True\" \"with_rtsp\" => \"True\" \"with_smb\" => \"True\" \"with_smtp\" => \"True\" \"with_ssl\" => \"openssl\" \"with_symbol_hiding\" => \"False\" \"with_telnet\" => \"True\" \"with_tftp\" => \"True\" \"with_threaded_resolver\" => \"True\" \"with_unix_sockets\" => \"True\" \"with_verbose_debug\" => \"True\" \"with_verbose_strings\" => \"True\" \"with_zlib\" => \"True\" \"with_zstd\" => \"False\"]"}
```

Comment:
This syntax for an options k/v array currently isn't read correctly by the lock file parsing code.